client/src/components/maps/MyMap.js

```diff
@@ -60,6 +69,9 @@ export const MyMap = () => {
   //   }
   // }
 
+  console.log(pins[0])
```

Comment:
Is this printing anything on the webpage?

Reply:
removed, looks like it was just for testing